### PR TITLE
Update broken links in function.ipynb

### PR DESCRIPTION
**Pull request by synandi**
Fixes https://github.com/tensorflow/tensorflow/issues/60459
Fix broken links in lines 64, 441 and 975

### CODE REVIEW

**Review by sachinprasadhs — APPROVED** [review]

File: site/en/guide/function.ipynb [hunk 1]
```diff
@@ -1832,7 +1832,7 @@
       "source": [
         "## Further reading\n",
         "\n",
-        "To learn about how to export and load a `Function`, see the [SavedModel guide](../../guide/saved_model). To learn more about graph optimizations that are performed after tracing, see the [Grappler guide](../../guide/graph_optimization). To learn how to optimize your data pipeline and profile your model, see the [Profiler guide](../../guide/profiler.md)."
+        "To learn about how to export and load a `Function`, see the [SavedModel guide](https://github.com/tensorflow/docs/blob/master/site/en/guide/saved_model.ipynb). To learn more about graph optimizations that are performed after tracing, see the [Grappler guide](https://github.com/tensorflow/docs/blob/master/site/en/guide/graph_optimization.ipynb). To learn how to optimize your data pipeline and profile your model, see the [Profiler guide](https://github.com/tensorflow/tensorboard/blob/master/docs/tensorboard_profiling_keras.ipynb)."
```
Comment (MarkDaoust) [inline site/en/guide/function.ipynb:1835]:
```suggestion
        "To learn about how to export and load a `Function`, see the [SavedModel guide](saved_model.ipynb). To learn more about graph optimizations that are performed after tracing, see the [Grappler guide](graph_optimization.ipynb). To learn how to optimize your data pipeline and profile your model, see the [Profiler guide](profiler.md)."
```

**Review by MarkDaoust — CHANGES_REQUESTED** [review]

**Comment by MarkDaoust** [comment]
Thanks for the fixes. 

Please prefer relative links where possible, and tensorflow.org links where relative links are not possible.

**Review by 8bitmp3 — APPROVED** [review]

**Comment by 8bitmp3** [comment]
@synandi Can you check the CLA and sign it (link to details: https://github.com/tensorflow/docs/pull/2223/checks?check_run_id=15147043836 itis the same as the "Details" link below where it says "cla/google  Failing.... Missing CLA...").

**Comment by gbaned** [comment]
Closing this PR due to changes are merged through [PR#2269](https://github.com/tensorflow/docs/pull/2269)  Thank you!

**Comment by 8bitmp3** [comment]
> Please close this PR because changes are merged through https://github.com/tensorflow/docs/pull/2269 Thank you!
@gbaned :+1:

**Pull request closed without merging**